src/python/WMCore/ReqMgr/Service/Auxiliary.py

```diff
@@ -186,10 +195,41 @@ def post(self):
             raise MissingPostData()
         else:
             doc = json.loads(data)
-        doc = Document(self.name, doc)
+        if subName:
+            docName = "%s_%s" % (self.name, subName)
+        else:
+            docName = self.name
+        doc = Document(docName, doc)
         result = self.reqmgr_aux_db.commitOne(doc)
         return result
 
+    @restcall(formats=[('text/plain', PrettyJSONFormat()), ('application/json', JSONFormat())])
+    def put(self, subName=None):
```

Comment:
Hmm, I didn't see previous reply for this. (I am pretty sure I did).  
The problem is there is "_"  prefix from subName so even if we have empty sting as default we still need a if else statement.